### ISSUE TITLE
[Slideshow Block] The slides settings are not working properly

### ISSUE THREAD

**Issue by andreiungurianu** [pixelgrade/nova-blocks#202]
When I visit the "Slides" tab on a new Slideshow block, I notices two issues:

- [ ] I don't see the preview of the images

- [ ]  The dimensions helper is not working

Here's a helpful video:

[Screen Recording 2020-08-24 at 02.16.30 PM.mp4.zip](https://github.com/pixelgrade/nova-blocks/files/5117423/Screen.Recording.2020-08-24.at.02.16.30.PM.mp4.zip)

I'm using Rosa 2 v1.3.0, Nova Blocks v1.5.2 and Wordpress 5.5

**Comment by madalingorbanescu** [comment]
Hey @andreiungurianu,

After doing some digging, I've found the reason for this issue.
We were not using any fallback for the image sizes used there, and we were using only the image sizes that should have been created by Nova Blocks. Unfortunately, those image sizes were not created so there was nothing to be shown. I've improved the code and added WordPress default image sizes as a fallback so when Nova Blocks image sizes are not created, the default ones from WordPress will be used.

But the main culprit for this issue was not our plugin, but [Local](https://localwp.com/), the tool that we are using to develop WordPress locally. The issue was that PHP 7.4.1 was missing a specific library. 

I am going to post the links that helped me to solve the problem, in case anyone will encounter this problem.

[What was causing the problem?](https://localwp.com/community/t/imagemagick-in-local-5-x/16333)
[The solution](https://localwp.com/community/t/how-can-i-manually-update-downloadable-lightning-services/21149)
